app/views/layouts/aeon/_prelim.html.erb

```diff
@@ -54,6 +54,7 @@
 						</ol>
 					</div>
 				</div>
+				<%= hidden_field_tag :authenticity_token, form_authenticity_token -%>
```

Comment:
Could you have alternatively used the rails form helper (https://guides.rubyonrails.org/form_helpers.html) to set the authenticity_token automatically? If there's some reason you can't (for example, this is an urgent issue and you didn't want to expand the scope of this fix with a refactor), seems like this should also work!

Reply:
I do think that's a better long-term approach. Refactoring the Aeon forms will take some time, though, and I'd say this issue is urgent enough to warrant the less elegant fix.